.travis.yml

```diff
@@ -2,28 +2,24 @@ language: java
 
 matrix:
   include:
-    - os: linux
+    - name: Ubuntu 14.04 JDK 9
+      os: linux
       jdk: oraclejdk9
```

Comment:
Shall we change this back to oraclejdk8, which is the Java compatibility we should we using (as per Gradle and our DeveloperGuide)

Reply:
I believe in #530, @eugenepeh wants us to move to JDK11.

As we target `JDK8` in Gradle, the Jar should be JDK8 compliant. 🤔

Reply:
Yes we should plan to move to JDK11 soon. But currently we are still at JDK8, so our tests environment should be running in JDK8 as well.

Reply:
Switching of JDK versions should be done in another PR? 
Since it involves AppVeyor too as it is using JDK9 `JAVA_HOME: C:\Program Files\Java\jdk9`.

Reply:
>Switching of JDK versions should be done in another PR?
Since it involves AppVeyor too as it is using JDK9 `JAVA_HOME: C:\Program Files\Java\jdk9`.

Hmm, yea probably can raise a different issue for that